xml_inscriptions/transcribed/IN.Bloom.IUAM.G.63.105.84a.xml

```diff
@@ -25,15 +25,15 @@
                <msContents>
                   <textLang mainLang="grc" otherLangs=""/>
                   <msItem class="#magic">
-                     <p>Male Deity with Scepter and Situla</p>
+                     <p>Magic inscription</p>
                   </msItem>
                </msContents>
                <physDesc>  <!--  *************************************  <objectDesc>  *********************************  -->
                   
                   <objectDesc ana="#object_other">
                      <supportDesc ana="#material_other">
                         <support>
-                           <p> dark brown agate with red striations</p>
+                           <p>Agate stone</p>
```

Comment:
Any particular reason you're backing away from the more precise description of the material here?

Reply:
Prof. Bodel mentioned that this might be too much detail for the description of the support, which is why I want back elsewhere and took out other descriptors for the sort of marble, but I don't mind adding them back

Reply:
OK, if he wants to strip out some of this information that's fine; we'll probably want to have a conversation down the road about whether we want to reincorporate it, since some of it is likely good to have. If nothing else, Git will save the previous versions of the files for us so we won't be losing the data forever, if we remove it now.

Reply:
I've gone ahead and marked all the instances of this that were still open as resolved; no need to take action on these specifically.